stylesheets/grid/modern/_grid.styl

```diff
@@ -98,6 +98,9 @@
       margin-top: $sizing-400
 
   @media $media-xlarge
+    &--3
```

Comment:
I've been using it as g--3 = 3 items per row. g--4 = 4 items per row. etc. It's not completely intuitive, but matches what was setup originally for Boston.gov.